code/modules/projectiles/guns/mounted.dm

```diff
@@ -89,7 +89,7 @@
 //-------------------------------------------------------
 //MG-2005 mounted minigun
 
-/obj/item/weapon/gun/minigun_nest
+/obj/item/weapon/gun/standard_minigun
```

Comment:
minor but why is this 'standard' minigun, when there is already a standard minigun?